homeassistant/components/camera/strings.json

```
@@ -106,6 +106,10 @@
           "description": "Planned lookback period to include in the recording (in addition to the duration). Only available if there is currently an active HLS stream. The actual length of the lookback period may vary."
         }
       }
+    },
+    "stop_record": {
+      "name": "Stop Recording",
+      "description": "Stops an on-going Recording."
```

Sentence-casing needed, "ongoing" without a hyphen, and little more text in the description:

```suggestion
      "name": "Stop recording",
      "description": "Stops an ongoing camera recording."
```

You might want to clarify even more.